packages/building-webpack/src/modern-web-webpack-plugin.js

```diff
@@ -54,6 +54,11 @@ const legacyError = error(
   'You are running webpack-dev-server with a legacy build, run the dev server with the --legacy flag for this browser or run a production build.',
```

Comment:
I think the solution is correct but I am not sure if we need to apply it in all places. I think we only need this in places where we ship to the browser, right? Any code that runs in mode still needs os specific separators. Are you sure we got this covered correctly?

Reply:
I tested this code on Windows 7 just now and it works ;).

Reply:
all the pathes changed are only used to finally end up in the frontend... so I don't see a path where we should not have applied it?

Reply:
Ok, thanks for testing. I was reviewing on my phone so it was hard to tell.